src/ioctls/system.rs

```diff
@@ -583,6 +640,12 @@ mod tests {
         Kvm::new().unwrap();
     }
 
+    #[test]
+    fn test_kvm_new_with_path() {
+        let kvm_path = std::ffi::CString::new("/dev/kvm").unwrap();
```

Comment:
Why not test the same `cstr` build as the func?
```suggestion
        let kvm_path = unsafe { CStr::from_bytes_with_nul_unchecked(b"/dev/kvm\0") };
```

Reply:
Done.